pyiron_base/_tests.py

```diff
@@ -88,4 +89,52 @@ def tearDown(self):
         self.project.remove_jobs_silently(recursive=True)
 
 
-_TO_SKIP = [PyironTestCase, TestWithProject, TestWithCleanProject]
+class ToyJob(PythonTemplateJob):
+    def __init__(self, project, job_name):
+        """A toyjob to test export/import functionalities."""
+        super(ToyJob, self).__init__(project, job_name)
+        self.input['input_energy'] = 100
```

Comment:
```suggestion
        self.input.in_data = 100
```

`PythonTemplateJob` uses `DataContainer` for IO now. Also while 'energy' is pretty generic, since this is in base let's go all the way with our generality.